### ISSUE TITLE
Suggested feature: easy way to derive phase curve features of interest

### ISSUE THREAD

**Issue by mhooton** [bmorris3/kelp#25]
Right now it's quite easy to determine features of interest for the generated phase curves such as:
- min/max flux/amplitude
- integrated dayside flux (i.e. eclipse depth)/nightside flux
- conversion of hotspot offset to more traditional definition (more useful when omega_drag is <=3)
using a few extra lines of code. However, it might be nice to add a feature that will do this automatically, which might be especially desirable for new users. Perhaps a neat way of implementing this could be a new keyword in the each of the phase curve methods such as "derive". If set to True, it will return a dictionary of relevant values such as those listed above as an extra output.

**Comment by bmorris3** [comment]
Hi @mhooton!

> min/max flux/amplitude

This operation can be accomplished with the following syntax: 
```python
Model(...).thermal_phase_curve(...).flux.max()  # or .min()
```
> conversion of hotspot offset to more traditional definition (more useful when omega_drag is <=3)

This can be done with 
```python
xi[Model(...).thermal_phase_curve(xi, ...).flux.argmax()]  # radians
```
> integrated dayside flux (i.e. eclipse depth)/nightside flux

This one is a bit subtle since it might refer to either the reflected or thermal components, or both together. I think it might be cleaner to provide convenience functions like `Model.albedo_redist` as implemented in #33, and/or to ask the user to run something like: 
```python
flux_dayside = Model(...).thermal_phase_curve(xi=0,...).flux
flux_nightside = Model(...).thermal_phase_curve(xi=np.pi,...).flux
```
Are these suggestions ok by you? I realize it's convenient for me to suggest making no changes to the code, but I think it's clearer if the user actually asks explicitly for what they want from the code.

**Comment by mhooton** [comment]
That looks fine
